macros/generic_tests/test_exists.sql

```diff
@@ -37,7 +49,13 @@ where "INFORMATION_SCHEMA"."COLUMNS"."TABLE_SCHEMA" = '{{ model.schema }}'
         {% else %}
             {% set log_category = 'UserError' %}
         {% endif %}
-        {{ log(tojson({'Key': 'TestExists', 'Details': {'model_name': model.name, 'column_name': column_name}, 'Category': log_category, 'Message': 'The field \'' ~ model.name ~ '.' ~ column_name ~ '\' doesn\'t exist in the source data. Note that the field detection is case-sensitive.'}), True) }}
+        {% if column_name != null %}
+            {% if table_exists %}
+                {{ log(tojson({'Key': 'TestExists', 'Details': {'model_name': model.name, 'column_name': column_name}, 'Category': log_category, 'Message': 'The field \'' ~ model.name ~ '.' ~ column_name ~ '\' doesn\'t exist in the source data. Note that the field detection is case-sensitive.'}), True) }}
```

Comment:
The key for both messages is the same, while these should be unique for each message type. In the future these keys in combination with the details will be used for translations.

Reply:
Key has been updated so both are unique, also gave the existing key a better name, the keys aren't used yet